rangeMessages.js

```diff
@@ -0,0 +1,104 @@
+const langaugeMessages = {
+  temperature: {
+    dataRanges: [
+      {
+        limit: Number.MIN_SAFE_INTEGER,
+        English: "Out of Bound",
```

Comment:
Consider that we are adding another language, like Spanish. The developer may miss out adding the `Spanish` key in one of these objects. Think of a test to catch that mistake

Reply:
Ok sir